core/server/data/migration/fixtures/007/01-add-themes-permissions.js

```diff
@@ -0,0 +1,33 @@
+var utils = require('../utils'),
+    permissions = require('../../../../permissions'),
+    resource = 'theme';
+
+function getPermissions() {
+    return utils.findModelFixtures('Permission', {object_type: resource});
```

Comment:
yeah i slightly ignored that, because the logic behind works.
Tell me if you think that we need to change it 👍